content/projects/jcasc/dev-tools.adoc

```diff
@@ -67,6 +67,14 @@ For more details, see this Google Doc -  https://docs.google.com/document/d/1aPf
   * Wednesdays every two weeks.
   * Time - 9am (UTC+1)
   * link:https://docs.google.com/document/d/1FkJyU_-QqQeQV8h3UdGeT5IitWWVaqIIgsPgCoVoB6k/edit?usp=sharing[Meeting Notes]
+  
+=== JCasC VScode Plugin 
+
+  * link:https://marketplace.visualstudio.com/items?itemName=jcasc-developers.jcasc-plugin[VSCode MarketPlace Link]
+  * link:https://github.com/jenkinsci/jcasc-vscode-extension[Github Project Link]
+  
+=== Blogs
+  * link:https://jenkins.io/blog/2019/10/08/jcasc-phase1-blog/[VSCode Blog]
```

Comment:
Since the blog is inside the jenkins.io site, this link should be relative to the site.  Refer to content/sigs/cloud-native/index.adoc and content/sigs/cloud-native/pluggable-storage.adoc for two examples of the technique
```suggestion
  * link:/blog/2019/10/08/jcasc-phase1-blog/[VSCode Blog]
```